Makefile

```diff
@@ -43,38 +43,6 @@ generate:
 manifests:
 	$(MAKE) -f kubebuilder.mk manifests
 
-# Generate protobufs
```

Comment:
Please remove `make proto` from line 39 and update the corresponding comments as well.

Reply:
We can also remove PROTO_IMG definition from this file.